src/main/java/by/bookify/auth/controller/AuthenticationController.java

```diff
@@ -12,26 +12,29 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import static by.bookify.user.model.Role.ADMIN;
+import static by.bookify.user.model.Role.USER;
+
 @RestController
-@RequestMapping("/auth")
+@RequestMapping("/v1/auth")
 @RequiredArgsConstructor
 public class AuthenticationController {
 
     private final AuthService authService;
 
     @PostMapping("/register")
     public ResponseEntity<AuthResponseDto> registerUser(@RequestBody RegisterRequestDto registerRequestDto) {
-        return ResponseEntity.ok(authService.registerUser(registerRequestDto));
+        return ResponseEntity.ok(authService.registerUser(registerRequestDto, USER));
     }
 
     @PostMapping("/register/admin")
     @PreAuthorize("hasRole('SUPER_ADMIN')")
     public ResponseEntity<AuthResponseDto> registerAdmin(@RequestBody RegisterRequestDto registerRequestDto) {
-        return ResponseEntity.ok(authService.registerAdmin(registerRequestDto));
+        return ResponseEntity.ok(authService.registerUser(registerRequestDto, ADMIN));
```

Comment:
Renaming proposal to register(). Cause registerUser with role Admin can be confusing

Reply:
Exacltly.
Fixed